lighthouse-core/formatters/partials/critical-request-chains.html

```diff
@@ -55,6 +55,12 @@
   color: #999;
 }
 
+.initial-nav {
```

Comment:
we'll need to add this to the .css file when https://github.com/GoogleChrome/lighthouse/pull/1652 lands.